AdminWebsite/AdminWebsite/Services/UserAccountService.cs

```diff
@@ -216,27 +217,33 @@ public async Task DeleteParticipantAccountAsync(string username)
         public async Task AssignParticipantToGroup(string username, string userRole)
         {
             const string REPRESENTATIVE_ROLE = "Representative";
-            
+            const string JOH_ROLE = "Judicial Office Holder";
+
             // Add user to user group.
-            var addUserToGroupRequest = new AddUserToGroupRequest
+            await AddGroup(username, External);
+            switch (userRole)
             {
-                User_id = username,
-                Group_name = External
-            };
+                case REPRESENTATIVE_ROLE:
+                    await AddGroup(username, VirtualRoomProfessionalUser);
+                    break;
+                case JOH_ROLE:
+                    await AddGroup(username, JudicialOfficeHolder);
```

Comment:
@valentinabennett  have you checked the hearingscontroller > BookNewHearing actually sends the correct user role? Please add a test around that too.  Please check usernameAdIdDict  has the correct user role.

Reply:
usernameAdIdDict has not user role, it just keep Id of account in AAD and user name. The user-role we have from participant details ( we read booking by id)

Reply:
participant.User_role_name

Yes, you are right.